zerver/decorator.py

```diff
@@ -45,6 +47,19 @@
 FuncT = TypeVar('FuncT', bound=Callable[..., Any])
 ViewFuncT = TypeVar('ViewFuncT', bound=Callable[..., HttpResponse])
 
+## logger setup
+log_format = "%(asctime)s: %(message)s"
+logging.basicConfig(format=log_format)
```

Comment:
Is this `basicConfig` line required?  I don't think we use it our other loggers that run inside the main process (as opposed to in a queue worker)